dist/images/ovnkube.sh

```diff
@@ -109,9 +110,44 @@ ovn_controller_opts=${OVN_CONTROLLER_OPTS:-"--ovn-controller-log=-vconsole:info"
 ovnkube_loglevel=${OVNKUBE_LOGLEVEL:-4}
 
 
+net_cidr=${OVN_NET_CIDR:-10.128.0.0/14/23}
+svc_cidr=${OVN_SVC_CIDR:-172.30.0.0/16}
 
 # =========================================
 
+# Get the ovn master host
+# By default the cluster comes up with 3 masters and the ovn master runs on one of the masters.
+# The ovn master is run from a deployment with one replica and the deployment is exposed in a service.
+# The exposed entrypoint (when the master is running) is the host running the ovn-master.
+get_ovn_master_host () {
+  # get api server
+  k8s_server=$(KUBECONFIG=/etc/origin/node/node.kubeconfig kubectl config view --minify | gawk '/server:/{ print $2 }')
+
+  ovn_svc_host_fqn=$(echo ${k8s_server} | sed 's;//;;' | gawk -F : '{ print $2 }')
+  ovn_svc_host=$(host ${ovn_svc_host_fqn} | gawk '{print $4}')
```

Comment:
ovn_svc_host and ovn_svc_host_fqn don't look like they get used anywhere?

Reply:
deleted lines